ulib/FStar.ReflexiveTransitiveClosure.fsti

```diff
@@ -30,26 +30,42 @@ module FStar.ReflexiveTransitiveClosure
 ///
 /// See examples/preorder/Closure.fst for usage examples.
 
-open FStar.Preorder
+let binrel (a:Type) = a -> a -> Type
```

Comment:
Just uses binrel in WellFounded, Lex, and this module, so that we're properly doubly universe polymorphic